src/QMCHamiltonians/MomentumEstimator.h

```diff
@@ -77,19 +75,16 @@ class MomentumEstimator: public QMCHamiltonianBase
   std::vector<int> kWeights;
   ///dims of a grid for k points
   int kgrid;
+  ///maximum k-value in the k-grid in cartesian coordinates
+  RealType kmax;
```

Comment:
If they are only used inside putSpecial, move them there.

Reply:
Currently this is true. For post-processing purposes it could be good to include more information in the h5-files, at some point.

Reply:
I think even when you write them to h5, these meta data need to be written in putSpecial.